firefox/background.entry.js

```diff
@@ -314,6 +314,15 @@ addListener('addURLToHistory', url => {
 	});
 });
 
+addListener('isURLVisited', url =>
+	new Promise(resolve => {
+		/* eslint no-unused-vars: ["error", { "args": "none" }] */
```

Comment:
Are you sure this is needed? ESLint should allow unused args as long as they're followed by used args